.github/workflows/post_release.yml

```diff
@@ -0,0 +1,70 @@
+name: post_release
+on:
+  workflow_run:
+    workflows: ["release"]
+    types:
+      - completed
+
+jobs:
+  post_release:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Cache cargo & target directories
+        uses: Swatinem/rust-cache@v2
+        with:
+          key: "v2"
+      - name: Install musl-tools, gnome-keyring and keyutils on Linux
+        run: |
+          sudo apt-get update --yes && sudo apt-get install --yes musl-tools gnome-keyring keyutils
+          rm -f $HOME/.local/share/keyrings/*
+          echo -n "test" | gnome-keyring-daemon --unlock
+      - name: Build binary
+        uses: houseabsolute/actions-rust-cross@v0
+        with:
+          command: "build"
+          target: x86_64-unknown-linux-musl
+          args: "--verbose --release --package bench"
+      - name: Spin up Docker Container
+        run: docker run -d -p 8090:8090 --name iggy_container iggyrs/iggy
+      - name: Test Benchmark
+        run: |
+          send_output=$(timeout 1s ./target/x86_64-unknown-linux-musl/release/iggy-bench send tcp --server-address 127.0.0.1:8090 2>&1) || send_code=$?;
+          poll_output=$(timeout 1s ./target/x86_64-unknown-linux-musl/release/iggy-bench poll tcp --server-address 127.0.0.1:8090 2>&1) || poll_code=$?;
+          send_poll_output=$(timeout 1s ./target/x86_64-unknown-linux-musl/release/iggy-bench send-and-poll tcp --server-address 127.0.0.1:8090 2>&1) || send_poll_code=$?;
+
+          expected_send_lines=("Producer #1 → preparing the test messages..."
+                               "Producer #1 → sending 1000000 messages in 1000 batches of 1000 messages...")
+          expected_poll_lines=("Consumer #1 → preparing the test messages..."
+                               "Consumer #1 → polling 1000000 messages in 1000 batches of 1000 messages...")
+          expected_send_poll_lines=("${expected_send_lines[@]}"
+                                    "${expected_poll_lines[@]}")
+
+          # Check for successful timeout
+          exit_codes=($send_code $poll_code $send_poll_code)
+          for code in "${exit_codes[@]}"; do
+            if [ "${code:-0}" -ne 124 ]; then
```

Comment:
Please add in the comment why there's check for 124

Reply:
Done!